travis.sh

```diff
@@ -75,7 +77,8 @@ if [[ "$ROS_DISTRO" == "kinetic" ]] && ! [ "$IN_DOCKER" ]; then
       -e USE_DEBROS_DISTRO \
       -v $(pwd):/root/ci_src industrial-ci/xenial \
       /bin/bash -c "cd /root/ci_src; source .ci_config/travis.sh;"
-  exit 0
+  travis_time_end  # run_travissh_docker
+  exit 0  # TODO; Need to return approproate values based on the test result.
```

Comment:
I assume you meant `travis run` --> `docker run` and `set +e` --> `set -e`? If so,

> this entire script will exit if that value is non-zero. So the only way to reach the exit 0 is if everything is OK.

Yes.
I guess my inline comment was misleading, so removed it. Hope the updated code now makes sense.